sw/device/silicon_creator/lib/attestation.h

```diff
@@ -75,6 +75,18 @@ typedef enum {
    * The CDI_1 attestation key seed.
    */
   kCdi1AttestationKeySeed = 2,
+  /**
+   * The TPM EK attestation key seed.
+   */
+  kTpmEkAttestationKeySeed = 3,
```

Comment:
We will need to update the provisioning firmware to provision these seeds into a flash info page during manufacturing. We can put them on the same page as the attestation key seeds if we would like. The current provisioning firmware only provisions the attestation key seeds [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/lib/personalize.c;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=373). These seeds are loaded from flash info into OTBN DMEM during key generation by `otbn_boot_attestation_keygen`  [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/otbn_boot_services.c;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=141).

I can turn this into a separate issue, and we can address in a follow up PR.